include/openthread/types.h

```diff
@@ -743,6 +743,16 @@ typedef struct otExternalRouteConfig
      * TRUE, if this configuration is considered Stable Network Data.  FALSE, otherwise.
      */
     bool mStable : 1;
+
+    /**
+     * TRUE if the external route entry's next hop is the device itself (i.e., the route was added by device itself).
+     * FALSE otherwise.
+     *
+     * This value is ignored when adding an external route. For any added route the next hop is by default the device
```

Comment:
Change: "next hop is by default the device itself" to "next hop is this device".